### ISSUE TITLE
Add direct Google Maps and / or OSM shortcut to context menu

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#9787]
The whole copy cursor geo position thing still needs too many clicks. Maybe we can add a configurable URL template which the user can change via GUI settings

**Comment by angelobanse** [comment]
Or even better, a shortcut to GeoHack (e.g. https://geohack.toolforge.org/geohack.php?params=52_31_17_N_13_24_37_E), just like Wikipedia does with coordinates.

This requires converting the coordinate to DMS.

**Comment by behrisch** [comment]
Good idea! And actually it does not require conversion, decimal degrees work just fine: https://geohack.toolforge.org/geohack.php?params=52.31;13.24

**Comment by behrisch** [comment]
@namdre @angelobanse Do we still want to have this configurable to save one more click?

**Comment by namdre** [comment]
Yes

**Comment by namdre** [comment]
And the configuration should persists in the fox registry.  (I don't want it in a gui settings file since that might be a security issue).

**Comment by angelobanse** [comment]
Should the configured service replace GeoHack - or would it be an additional option in the context menu?

**Comment by namdre** [comment]
I think it should replace geohack with a menu entry "show curser geo-position in web map"
- the Application Settings dialog could be used to configure the template string.
- The default could be `https://geohack.toolforge.org/geohack.php?params=LAT;LON`
- Calling the menu entry would replace LAT;LON with the respective values.
- The user could then change the template to something like `https://www.google.com/maps?ll=LAT,LON`

**Comment by behrisch** [comment]
@angelobanse Could you take care of

- [ ] make the table in the AppSettingsDialog more beautiful
- [ ] extend the list of defaults if useful, I currently use: https://www.google.com/maps?ll=%lat,%lon&t=h&z=18 and https://www.openstreetmap.org/?mlat=%lat&mlon=%lon&zoom=18&layers=M

- [ ] document it

**Comment by behrisch** [comment]
@angelobanse The name GoogleSat was there for a reason because it is actionally not only the map we are linking to but the hybrid map/satellite view. Maybe GoogleMapsSat? But I also don't mind if we keep the new name because it correctly names the provider.

**Comment by angelobanse** [comment]
@behrisch let's keep it as "Google Maps". Satellite is only a view that can be changed with one click on the map.
